Project.toml

```diff
@@ -13,7 +15,9 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 LorentzVectorHEPStaticArraysExt = "StaticArrays"
 
 [compat]
+InteractiveUtils = "1.11.0"
```

Comment:
I think this should be the same version as julia itself

Reply:
As mentioned above, it is needed for `subtypes` to get all supported coordinate systems.